### DISCUSSION TITLE
RSS v1.0 support?

### DISCUSSION THREAD

**Discussion by creepy-pasta101** [spacecowboy/Feeder#181] [Q&A]
Does the app support RSS v1.0? I had problems with BioRxiv feed `http://connect.biorxiv.org/biorxiv_xml.php?subject=all` 
And yes... they provide it with wrong content type with header.. but on continuing anyway it isn't parsed...

**Comment by spacecowboy** [comment]
it's not an rss feed. it's an RDF document.
